Dockerfile

```diff
@@ -18,6 +18,10 @@ RUN mkdir -p /root/.local/bin && \
     wget -q -O- $STACK_DOWNLOAD_URL | tar --strip=1 -xvz -C /root/.local/bin/ && \
     chmod +x /root/.local/bin/stack
 
+# Install Rust FFI dependencies and build link target
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y
+COPY notifier notifier
```

Comment:
@BrianHicks what is this doing?

Reply:
there's no source copied at this point, so we copy the notifier source and build it. You reckon it should take place elsewhere?